module/os/linux/zfs/zvol_os.c

```diff
@@ -1110,7 +1110,11 @@ zvol_os_create_minor(const char *name)
 		rw_enter(&zvol_state_lock, RW_WRITER);
 		zvol_insert(zv);
 		rw_exit(&zvol_state_lock);
+		#ifdef HAVE_ADD_DISK_RET
+		error = add_disk(zv->zv_zso->zvo_disk);
+		#else
 		add_disk(zv->zv_zso->zvo_disk);
+		#endif
 	} else {
```

Comment:
Dang it, I thought I pushed the amended commit with those style nits fixed. Oh well, I'll include it before merge or when I next push.